lib/ui/login/login_screen.dart

```diff
@@ -173,15 +176,23 @@ class LoginScreenState extends ConsumerState<LoginScreen>
                   right: AppDimensions.spacing24,
                   bottom: AppDimensions.spacing20,
                 ),
-                child: _configuredTextField(context, true),
+                child: _configuredTextField(
+                  context,
+                  true,
+                  'emailTextFieldKey',
+                ),
               ),
               Padding(
                 padding: const EdgeInsets.only(
                   left: AppDimensions.spacing24,
                   right: AppDimensions.spacing24,
                   bottom: AppDimensions.spacing20,
                 ),
-                child: _configuredTextField(context, false),
+                child: _configuredTextField(
+                  context,
+                  false,
+                  'passwordTextFieldKey',
```

Comment:
Should we create a separate file to store all the keys? It will be nice to have an overview 🙏